starter-code/server.js

```diff
@@ -66,18 +83,28 @@ app.put('/articles/:id', function(request, response) {
   // an author_id property, so we can reference it from the request.body.
   // TODO: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors(author, "authorUrl")
+     VALUES ($1, $2)
+     WHERE author = $1`,
+    [request.body.author,
+      request.body.authorUrl]
   )
   .then(function() {
     // TODO: Write a SQL query to update an article record. Keep in mind that article records
     // now have an author_id, in addition to title, category, publishedOn, and body.
     // TODO: Add the required values from the request as data for the SQL query to interpolate
     client.query(
-      ``,
-      []
-    )
-  })
+      `UPDATE articles(author_id, title, category, "publishedOn", body)
+      SELECT author_id, $1, $2, $3, $4
+      FROM authors
+      WHERE author = $5;
+    `,
```

Comment:
Pretty much the same case here, you are not setting anything to your database. You want to do a SET ...WHERE